A50-xds-outlier-detection.md

```diff
@@ -0,0 +1,207 @@
+A50: gRPC xDS Outlier Detection Support
+----
+* Author(s): murgatroid99
+* Approver: markdroth
+* Implemented in: C++, Java, Go, Node
```

Comment:
It's not actually implemented in any languages yet, so we should probably not list them yet.

Reply:
Done.